README.md

```diff
@@ -102,6 +102,67 @@ $ yarn dev
 
 Open http://dev.localhost:3000/ and test away!
 
+## Docker
+```
+```
+
+</p>
+</details>
+
+---
+
+<details><summary><b>Project setup in Docker</b></summary>
+<p>
+
+## Ports used
+
+```
+5500 - Frontend (client)
+5501 - Backend (server)
+5502 - postgres
+5503 - redis
+```
+
+## Setup dofus lab in docker
+
+#### Setup
+
+```
+docker-compose -f "docker-compose.debug.yml" up -d --build
+```
+
+#### Initialize database
+
+```
+docker-compose -f "docker-compose.initdb.yml" run --rm initdb
+```
+
+Open http://dev.localhost:5500/ and test away!
```

Comment:
Should we just keep the client on 3000? I imagine no one will be using these concurrently in any case, and then I wouldn't have to remember two different ports.

Reply:
yeah, changing the port is really not a problem, but i wanted to keep the possibility to run it concurently (atlest in this initial phase when docker is not adopted properly)